lib/src/scss-shared/_nx-tag-variables.scss

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-tag {
```

Comment:
I wouldn't thing this wrapper would really be necessary

Reply:
> I wouldn't thing this wrapper would really be necessary

f9c3f0d866d5c4b14e813c354451b5bbf031765e